### ISSUE TITLE
Plugin not compatible with TypeDoc 0.24.x

### ISSUE THREAD

**Issue by WikiRik** [Gerrit0/typedoc-plugin-missing-exports#19]
Just wanted to report that this plugin is failing with TypeDoc 0.24.

The following error occurs;
```
TypeError: type.getSymbol is not a function
    at Object.reference (/workspaces/sequelize/node_modules/typedoc-plugin-missing-exports/index.js:68:37)
    at Object.reference (/workspaces/sequelize/node_modules/typedoc/dist/lib/models/types.js:115:32)
    at ReferenceType.visit (/workspaces/sequelize/node_modules/typedoc/dist/lib/models/types.js:42:36)
    at discoverMissingExports (/workspaces/sequelize/node_modules/typedoc-plugin-missing-exports/index.js:96:27)
    at onResolveBegin (/workspaces/sequelize/node_modules/typedoc-plugin-missing-exports/index.js:27:23)
    at triggerEvents (/workspaces/sequelize/node_modules/typedoc/dist/lib/utils/events.js:191:43)
    at triggerApi (/workspaces/sequelize/node_modules/typedoc/dist/lib/utils/events.js:167:13)
    at eventsApi (/workspaces/sequelize/node_modules/typedoc/dist/lib/utils/events.js:60:18)
    at Converter.trigger (/workspaces/sequelize/node_modules/typedoc/dist/lib/utils/events.js:389:13)
    at Converter.resolve (/workspaces/sequelize/node_modules/typedoc/dist/lib/converter/converter.js:207:14)
```

This occurs in the following code; https://github.com/Gerrit0/typedoc-plugin-missing-exports/blob/6707a03b28f116fd6dec98674d15f197f3e2f32c/index.ts#L112-L124

From the TypeDoc changelog;
`Removed internal ReferenceType.getSymbol, reference types no longer reference the ts.Symbol to enable generation from serialized JSON.`

**Comment by Gerrit0** [comment]
Yep... typedoc's improvements here break the hack this plugin has relied on... fixing will likely require a full reimplementation.

**Comment by WikiRik** [comment]
Thanks for the quick response! Take all the time you need to resolve this

**Comment by Gerrit0** [comment]
Well, it got even hackier, but it works! Published as 2.0.0
